azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/scheduledflowpage.vm

```diff
@@ -114,17 +114,27 @@
                 #else
                   <td>Not Applicable</td>
                 #end
-                <td><button type="button" data-toggle="modal" data-target="#executionOptions">show</button></td>
+                <td>
+                ## Changed the style of "Show" button to be consistent with other buttons.
+                  <button type="button" class="btn btn-sm btn-info" data-toggle="modal"
+                          data-target="#executionOptions-${velocityCount}">Show
+                  </button>
+                </td>
 
-                  <div class="modal fade" id="executionOptions" tabindex="-1" role="dialog" aria-labelledby="executionOptionsLabel">
+              ## Distinguished executionOptions of each schedule with unique count number to fix
```

Comment:
Is this comment necessary?

Reply:
Since we are not very familiar with the UI part, I think it's necessary to fully document the details so that others can understand the intention of the code change.